File: Core/Ecommerce.Domain/Entities/CartItem.cs
using Ecommerce.Domain.Common;
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace Ecommerce.Domain.Entities
{
	public class CartItem : AuditableEntity
	{
		public int Id { get; set; }
		
		public int ProductId { get; set; }
		
		[Column (TypeName = "varchar(255)")]
		public Guid UserId { get; set; }
		
		public int Quantity { get; set; }
	}
}

File: UI/ecommerce-ui/src/models/CartItem.ts
export class CartItem {
    id: number;
    productId: number;
    userId: string;
    quantity: number;

    constructor(
        id: number,
        productId: number,
        userId: string,
        quantity: number
    ) {
        this.id = id;
        this.productId = productId;
        this.userId = userId;
        this.quantity = quantity;
    }
}
